liaisons/opcf/collaboration-options.md

```diff
@@ -0,0 +1,38 @@
+There are several options how the development of the OPC UA Binding for the Web of Things can be managed:
+
+# As W3C Note
+W3C members setup an OPC UA Binding document as W3C Note. OPC UA experts can be involved for getting feedback.
+
+Pro
+* alligned with existing approaches such as with Modbus Binding
+
+Cons
+* no testing / conformance testing provided
+
+# Develop W3C Note, transformed it later to W3C REC
+A previos developed W3C Note about OPC UA Binding can be used or transformed to a REC. OPC UA experts can be involved for getting feedback. 
+
+Pro
+*  includes testing
+
+Cons
+* still no conformance testing provided
+* requires more time 
+
+
+# As W3C REC only
+W3C members setup an OPC UA Binding document as W3C Recommendation. OPC UA experts can be involved for getting feedback in that work. 
+
+Pro
+*  includes testing
+
+Cons
+* still no conformance testing provided
+
+# As OPC UA Companion Standard only
+OPCF Members setup an OPC UA Binding document as OPC UA Companion. W3C WoT experts can be involved for getting feedback.
+
+Pro
+*  includes testing / conformance
+*  follows the new de-coupling binding approache of the new WoT charter 
```

Comment:
Cons
* Binding is not under control of the WoT WG and will require coordination to ensure it follows a consistent template, document structure, is consistent with the TD specification, etc.

Reply:
ok, I will add this